src/sentry/api/issue_search.py

```diff
@@ -142,7 +142,15 @@ def convert_query_values(search_filters, projects, user, environments):
     :return: New collection of `SearchFilters`, which may have converted values.
     """
 
-    def convert_search_filter(search_filter):
+    def convert_search_filter(search_filter, organization):
+        if search_filter.key.name == "empty_stacktrace.js_console":
```

Comment:
The idea here is that you're preventing anyone who is not allowed to try the tag via the UI; correct?

Reply:
yup!